test/writehero_test.cpp

```diff
@@ -294,8 +294,8 @@ static void AssertPlayer(Player &player)
 	ASSERT_EQ(CountBool(player._pSLvlVisited, NUMLEVELS), 0);
 	ASSERT_EQ(player._pNFrames, 20);
 	ASSERT_EQ(player._pWFrames, 8);
-	ASSERT_EQ(player._pAFrames, 0);
-	ASSERT_EQ(player._pAFNum, 0);
+	ASSERT_EQ(player._pAFrames, 18);
+	ASSERT_EQ(player._pAFNum, 10);

```

Comment:
why

Reply:
Because test fails without it. Player is assigned frames in town now

Reply:
I don't see any change related to it in this PR

Reply:
mistake then